xbmc/interfaces/Builtins.cpp

```diff
@@ -167,6 +168,7 @@ const BUILT_IN commands[] = {
   { "CancelAlarm",                true,   "Cancels an alarm" },
   { "Action",                     true,   "Executes an action for the active window (same as in keymap)" },
   { "Notification",               true,   "Shows a notification on screen, specify header, then message, and optionally time in milliseconds and a icon." },
+  { "textviewer",                 true,   "Opens TextViewer Dialog, specify header, then message." },
```

Comment:
btw we should use "OpenTextViewer" which makes it more clear what this cmd will do.